app/src/organisms/Desktop/RunProgressMeter/hooks/useRunProgressCopy.tsx

```diff
@@ -127,6 +131,23 @@ export function useRunProgressCopy({
       }
     }
   })()
+  const {reportModuleCommand} = useModuleCommandAnalytics()
+  // Check if error occurred, then determine type
+  let error; 
+  if (typeof runCommandDetails?.data?.error === 'object') {
+    error = runCommandDetails?.data?.error?.errorType;
+  }
+
+  reportModuleCommand({
+    runId: runId!,
+    action: runCommandDetails?.data?.commandType!,
```

Comment:
```suggestion
    action: runCommandDetails?.data?.commandType,
```
We shouldn't be asserting any types here. The interface either needs to handle `undefined` types or we need to nullish coalescence to a type.